Patch: 00_stolen_from_HEAD.patch
Author: Tollef Fog Heen <tfheen@debian.org>
Handle empty queue files (Closes: #212643)
Index: mailman-2.1.12/Mailman/Handlers/Scrubber.py
===================================================================
--- mailman-2.1.12.orig/Mailman/Handlers/Scrubber.py	2009-03-02 20:41:57.000000000 +0100
+++ mailman-2.1.12/Mailman/Handlers/Scrubber.py	2009-03-02 20:49:54.000000000 +0100
@@ -388,6 +388,8 @@
                     t = unicode(t, 'ascii', 'replace')
                 try:
                     # Should use HTML-Escape, or try generalizing to UTF-8
+                    if len(charset) == 0:
+                        charset = 'us-ascii'
                     t = t.encode(charset, 'replace')
                 except (UnicodeError, LookupError, ValueError,
                         AssertionError):
